src/views.py

```diff
@@ -5,18 +5,18 @@
 
 
 @router.post("/")
-def create_department(department: dict):
-    DepartmentManager.insert_data(data=department)
+def create(department: dict):
+    DepartmentManager.create(data=department)
     return {"Data added successfully"}
 
 
 @router.get("/", name="get_departments")
-def read_departments():
-    return DepartmentManager.get_info()
+def get_all():
+    return DepartmentManager.get_all()
 
 
 @router.get("/{dep_id}")
-def get_department(dep_id: int):
-    if not (department := DepartmentManager.get_id(dep_id)):
+def get_by_id(dep_id: int):
+    if not (department := DepartmentManager.get_by_id(dep_id)):
         raise HTTPException(status_code=404, detail="Departments id table not found")
```

Comment:
It's not in the scope of this PR, but fix the error message please: `Department with such id not found`